addons/stock/models/res_config_settings.py

```diff
@@ -69,6 +69,7 @@ def set_values(self):
         if not self.group_stock_multi_locations and location_grp in base_user.implied_ids and warehouse_grp in base_user.implied_ids:
             raise UserError(_("You can't desactivate the multi-location if you have more than once warehouse by company"))
 
+        previous = self.default_get([])
```

Comment:
```suggestion
        previous = self.default_get(['group_stock_multi_locations', 'group_stock_production_lot', 'group_stock_tracking_lot'])
```
Please at least restrict the `default_get` call to the necessary fields.

Reply:
I agreed, I updated the commit 😄